### PR TITLE
Add a sign-up form

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -12,7 +12,21 @@ layout: page
...
+		<p>The Slack is invitation-only, but invitations are freely granted. You can request one from any existing member or by filling out this form: </p>
+		<form action="http://api.wealljs.org/signup" method="POST">
+			<label><span>Name: </span><input name="name" required type="text" class="input-field"></label>
+			<label><span>Email: </span><input name="email" required type="email" class="input-field"></label>
+			<label><span>Twitter (optional): </span><input name="twitter" type="text" class="input-field"></label>
+			<label><span>GitHub (optional): </span><input name="github" type="text" class="input-field"></label>
+			<label><span>About You</span></label>
+			<textarea name="about" placeholder="Tell us a bit about yourself! Anything or nothing is fine!" class="textarea-field"></textarea>
+			<label><span>&nbsp;</span><input type="checkbox" name="coc" required  class="input-field"> I agree to the <a href="/code-of-conduct.ht
+			ml">Code of Conduct</a> and <a href="/enforcement.html">Enforcement Policy</a>
```
Comment (zkat) [inline index.md]: awkward line break mid-href here?

Comment (zkat) [inline index.md]: We don't have http://lgbtq.technology/enforcement.html. Are we planning on adding it?

File: index.md [hunk 2]
```diff
@@ -12,7 +12,21 @@ layout: page
...
+			</label>
+			<input type="hidden" name="redirect_uri" value="http://wealljs.org/postsignup">
```
Comment (zkat) [inline index.md]: We need a page for this in lgbtq.technology. It's just where wheelie will redirect after the signup gets sent, and it should just be a "ok we'll send you a slack invite soon" or so

File: index.md (hunk 1 above)
Comment (aredridel) [reply index.md]: WE SO SHOULD. But not in this commit.

Comment (aredridel) [reply index.md]: Fixed.

**Comment by zkat** [comment]
🐑 🎉

**Pull request merged**